apps/full-site-editing/full-site-editing-plugin/global-styles/src/no-support.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default ( { unsupportedFeature } ) => (
+	<p>{ __( "Your active theme doesn't support this feature: " ) + unsupportedFeature + '.' }</p>
```

Comment:
String concatenation like this isn't great for translators (think e.g. right to left languages). You should use `sprint` here: https://github.com/WordPress/gutenberg/tree/3a0a929f99ee9344b3cb3510008ab98aebeaaff2/packages/i18n#sprintf

See PCYsg-3hI-p2#partial-strings-and-concatenation

Reply:
Good catch! f62fc2f